server/settings.py

```diff
@@ -67,5 +67,10 @@ def env(variable, fallback_value=None):
 HIGHCHARTS_SERVER_HOST = env('HIGHCHARTS_SERVER_HOST', 'localhost')
 HIGHCHARTS_SERVER_PORT = env('HIGHCHARTS_SERVER_PORT', '6060')
 
+LANGUAGES = [
+    {'language': 'en-CA', 'label': 'English', 'source': True, 'destination': True},
```

Comment:
shouldn't it be `en_CA` and `fr_CA` ?

Reply:
I am not sure, it was as such in the ticket https://dev.sourcefabric.org/browse/SDCP-104

Reply:
and it's similar in ntb also https://github.com/superdesk/superdesk-ntb/blob/master/server/settings.py#L183

Reply:
Hi @petrjasek afaik we stick to hyphens, it's also used in newsml input/output, e.g. <language tag="en-US" />, and by Tansa